proxy/proxy.go

```diff
@@ -502,6 +502,7 @@ func mapRequest(r *http.Request, rt *routing.Route, host string, removeHopHeader
 		rr.Header = cloneHeader(r.Header)
 	}
 	rr.Host = host
+	rr.RemoteAddr = r.RemoteAddr
```

Comment:
@aryszka What do you think about this? If you don't agree, then I will try to do this only for `fastcgi` requests. It's important, because if `REMOTE_ADDR` is empty, the PHP backend cannot properly handle the other headers as well.

Reply:
I don’t think it will break something.

Reply:
for outgoing requests, this field is ignored, so it doesn't break anything, but TBH it would be nicer to solve it somehow for only the fastcgi, that relies on it and leave it blank for the rest of the logic. If possible.